app/index.js

```diff
@@ -395,7 +395,8 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       createWindow,
       updatePlugins: () => {
         plugins.updatePlugins({force: true});
-      }
+      },
+      getPlugins: plugins.getModules
```

Comment:
Because everywhere in plugin file, modules are mentioned. And `plugins.getPlugins` seems weird to me.
But you're right, it's really better to not mention `module` in exports.